components/slices/Article.vue

```diff
@@ -109,6 +110,9 @@ export default {
         this.content.article_background && this.content.article_background.url
       )
     },
+    id() {
+      return textToId(this.content.article_title[0].text)
```

Comment:
**thought:** dans ce commit, je me demande s'il faut considérer les cas où les titres ne sont pas présents, car il me semble que le titre n'est pas obligatoire partout. Dans ce cas, je pense que `this.content.article_title[0]` échouerait.